wagtail_transfer/static_src/chooser/PageChooserResult.js

```diff
@@ -99,7 +99,7 @@ class PageChooserResult extends React.Component {
           href={page.meta.html_url}
           target="_blank"
           rel="noopener noreferrer"
-          className="status-tag primary"
+          className="w-status w-status--primary"
```

Comment:
Thanks. I thought this was only for the status tag:
`{% status...`

But I guess you do need to search for the particular classname `status-tag`